pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/replication.tsx

```diff
@@ -7,10 +7,32 @@ import { Metric, Axis, AxisUnits } from "src/views/shared/components/metricQuery
 import { GraphDashboardProps, nodeDisplayName, storeIDsForNode } from "./dashboardUtils";
 
 export default function (props: GraphDashboardProps) {
-  const { nodeIDs, nodesSummary, storeSources } = props;
+  const { nodeIDs, nodesSummary, storeSources, tooltipSelection } = props;
 
   return [
-    <LineGraph title="Ranges" sources={storeSources}>
+    <LineGraph 
+      title="Ranges" 
+      sources={storeSources}
+      tooltip={(
+        <div>
+          {`Range information for ${tooltipSelection}:`}
+          <dl>
+            <dt>Ranges</dt>
+            <dd>Total number of ranges</dd>
+            <dt>Leaders</dt>
+            <dd>The number of ranges with leaders</dd>
+            <dt>Lease Holders</dt>
+            <dd>The number of ranges that have leases</dd>
+            <dt>Leaders w/o Lease</dt>
+            <dd>The number of Raft leaders without leases</dd>
+            <dt>Unavailable</dt>
+            <dd>The number of unavailable ranges</dd>
+            <dt>Under-replicated</dt>
+            <dd>The number of under-replicated ranges</dd>
```

Comment:
I propose dropping "The number of" from each of these, and editing them to be a little more precise:

- Ranges: All ranges
- Leaders: Ranges with a Raft leader
- Lease Holders: Ranges with a current lease
- Leaders w/o Lease: Ranges with a Raft leader without a lease
- Unavailable: Ranges unavailable due to node outage
- Under-replicated: Ranges temporarily under-replicated

This is just a start/suggestions, feel free to change whatever